File: MODUL-5/PYTHON/PRAK502-2310817220002-RAUDATULSHOLEHAH.py
def hitung (nilai1, nilai2):
    return abs(nilai1 - nilai2)

def mutlak(angka):
    if angka < 0:
        return -angka
    else:
        return angka

a, c, b, d = map(int, input().split())

hasil = hitung(a, b) + hitung(c, d)
print(mutlak(hasil))

File: MODUL-5/C/PRAK502-2310817220002-RAUDATULSHOLEHAH.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2)
{
    // Lengkapi Function ini
    return fabs(nilai1 - nilai2);
}

int mutlak(int angka)
{
    // Lengkapi Function ini
    if (angka < 0)
    {
        return -angka;
    }
    else
    {
        return angka;
    }
}

int main()
{
    int a, b, c, d;

    scanf("%d", &a);
    scanf("%d", &c);
    scanf("%d", &b);
    scanf("%d", &d);

    int Hasil = hitung(a, b) + hitung(c, d);
    printf("%d", mutlak(Hasil));

    return 0;
}